src/App.vue

```diff
@@ -8,11 +8,18 @@ import { useQuizStore } from "./store/QuizStore";
 const quizQuestions = ref(0);
 const questionsQueue = useQuizStore();
 const quizStarted = ref<boolean>(false);
+const quizMode = ref<string>("Tutor");
 
-const handleStartQuiz = (questionAmount: number) => {
+const handleStartQuiz = ({
+  questionAmount,
+  mode,
+}: {
```

Comment:
I would add the types in the `.d.ts` file

Reply:
Done